src/components/Switch/switch.component.css

```diff
@@ -0,0 +1,58 @@
+html {
+  font-size: 62.5%;
+}
+
+:root {
+  --main-color: #6200ee;
+  --secondary-color: #c39ef8;
+  --default-color: #fff;
+}
+
+.inputWrapper {
+  position: relative;
+  top: 0.5rem;
+  left: 0.5rem;
+}
+
+.slider {
+  display: flex;
+  cursor: pointer;
+  width: 7rem;
+  height: 3rem;
+  border-radius: 1.4rem;
+  background-color: #c6c6c6;
+  position: relative;
+  transition: background-color 0.2s;
+}
+
+.slider::before {
+  content: '';
+  position: absolute;
+  /* --"transition" no longer works if this is on and to and left are omitted*/
+  /* transform: translateY(-12%); */
+  top: -0.48rem;
+  left: -0.40rem;
+  width: 4rem;
+  height: 4rem;
+  border-radius: 9rem;
+  transition: 0.2s;
+  /* transition-duration: 900ms; */
+  background-color: var(--default-color);
+  box-shadow: 0rem 0.07rem 0.5rem 0.15rem rgba(88, 88, 88, 0.5);
+}
+
+/* changes the width of the slider after clicking on it */
+.slider:active:before {
+  width: 4.8rem;
+}

```

Comment:
```suggestion
```

This does look kind of cool, but it's not a requirement for the brief, so let's remove it to stay on scope.